src/utils/valueFixers.js

```diff
@@ -37,7 +37,7 @@ const toNumber = function (numberOrString) {
 
   let trueOrError = isNumberlike(numberOrString);
 
-  if (typeof trueOrError === `object`) {
+  if (typeof trueOrError instanceof Error) {
```

Comment:
I see it now! More needs to be changed here. `typeof Error === 'object'`, so when `isNumberlike()` returns an `Error`, this will come out false and won't behave the way we want.

Not sure I described that clearly. Feel free to ping me if I need to clarify.

Reply:
Ah I see now. The line should be written as `if (trueOrError instanceof Error)` since `typeof trueOrError` will always return `object` and `object instanceof Error` will always return `false`.

I can work on making that fix and resubmitting.